make.js

```diff
@@ -45,6 +45,7 @@ exports.makeCombinedAPI = function (apis, sourceDir, apiOutputDir) {
 
     customizations = parseDataFile("customizations.json");
     prereqsAndCleanupMap = parseDataFile("prerequisitesAndCleanup.json");
+    testTokenOverrideMap = parseDataFile("testTokenOverrides.json");

```

Comment:
>testTokenOverrides [](http://example.com/codeflow?start=42&length=18)

maybe "CallingEntityOverrides" is clearer?  Wasn't sure what this was until I looked more closely at the code